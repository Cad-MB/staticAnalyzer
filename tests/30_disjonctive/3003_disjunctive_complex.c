{
    int x;
    int y;
    x = 0;
    if (x < 5) {
      y = 10;
    } else {
      y = 20;
    }
    if (y >= 15) {
      x = 10;
    } else {
      x = 5;
    }
    print_all;
  }