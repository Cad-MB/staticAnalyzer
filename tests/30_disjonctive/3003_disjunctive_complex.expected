tests/30_disjunctive/3003_disjunctive_complex.c:14.2-12: [ x in {[5;5] ∨ [10;10]}, y in {[10;10] ∨ [20;20]} ]
analysis ended